src/tx/recovery/logrecord.rs

```diff
@@ -35,8 +42,15 @@ pub enum TxType {
     START = 1,
     COMMIT = 2,
     ROLLBACK = 3,
-    SETI32 = 4,
-    SETSTRING = 5,
+    SETI8 = 4,
+    SETU8 = 5,
+    SETI16 = 6,
+    SETU16 = 7,
+    SETI32 = 8,
+    SETU32 = 9,
+    SETSTRING = 10,
+    SETBOOL = 11,
+    SETDATE = 12,
```

Comment:
memo: Backward compatibility is lost.
Conventional database files can no longer be used.